docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,107 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_[Feynman Zhou](https://twitter.com/FeynmanZhou), CNCF Ambassador, ACR Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the ORAS CLI recently. ORAS v0.14 introduces four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+## What's new in ORAS 0.14
+
+<figure>
+  <img src="/blog/oras-0.14-and-future/carbon.svg" />
+</figure>
+
+
+Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.14.0) for details.
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provides new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a validated artifact registry to a personal conintainer registry, then attach a SBOM to it and discover the reference in a tree graph. 
```

Comment:
should we change "validated artifact registry" to "public registry"? I'm not sure what "validated" implies. Similiarly, personal, might be changed to private registry. You've also used "artifact registry" for the source, and container registry for the private version:

How about: 

> This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a public registry to a personal registry, then attach a SBOM to it and discover the reference in a tree graph.

Reply:
Yes. I was actually supposing all images in MAR are validated by Microsoft. That's the reason why we choose to copy the images from MAR as a trusted source.

How about:

> This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a public registry validated by Microsoft to a private registry, then attach an SBoM to it and discover the reference in a tree graph.